modules/specialized-guides/pages/salt/salt-connectivity.adoc

```diff
@@ -0,0 +1,156 @@
+[[salt-connectivity]]
+= Salt Connectivity
+
+Depending on the size and spreadness of the environment where {productname} is used some connectivity issues are possible.
+There are no common recommendations for all the possible use cases as the environments could be very different expecially if public clouds instances are involved.
+
+
+
+[[minions-connectivity]]
+== Minions Connectivity
+
+In case if clients are losing the connection to the {productname} Server (or {productname} Proxy if involved),
+they are unreachable from the {productname} Server web UI or using the CLI.
```

Comment:
```suggestion
In case clients are losing the connection to the {productname} Server (or {productname} Proxy if involved), they are unreachable from the {productname} Server {webui} or using the CLI.
```